README.md

```diff
@@ -71,6 +71,10 @@ When elements are extracted from PDFs or images, it may be useful to get their b
 	`pyenv  virtualenv 3.8.15 document-processing` <br />
 	`pyenv activate document-processing`
 
+  - To install the `detectron2` model, which
+  `unstructured` uses for layout detection:
+    - make install-detectron2
```

Comment:
nit: think this block fits more naturally combined with " Run make install" line a few lines below. I.e., "if image and high resolution pdf extraction is required, also run `make install-detectron2`"

... and actually ...

the default image layout model probably won't be detectron2, so i'd name it something more abstract to begin with.